sys/kern/subr_intr.c

```diff
@@ -327,12 +342,17 @@ isrc_release_counters(struct intr_irqsrc *isrc)
  *  from the assembler, where CPU interrupt is served.
  */
 void
-intr_irq_handler(struct trapframe *tf)
+intr_irq_handler_root(struct trapframe *tf, uint32_t rootnum)
 {
 	struct trapframe * oldframe;
 	struct thread * td;
+	struct intr_irq_root *root;
 
-	KASSERT(irq_root_filter != NULL, ("%s: no filter", __func__));
+	KASSERT(rootnum < INTR_ROOT_NUM,
+		("%s: invalid interrupt root %d", __func__, rootnum));
```

Comment:
FreeBSD style says this should be "\<tab\>\<space\>\<space\>\<space\>\<space\>".

Reply:
done